k8s/k8s.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/projectcalico/cni-plugin/utils"
 	"github.com/projectcalico/libcalico-go/lib/api"
 	k8sbackend "github.com/projectcalico/libcalico-go/lib/backend/k8s"
+	backendConverter "github.com/projectcalico/libcalico-go/lib/converter"
```

Comment:
I think in general camelcase for imports is weird (you probably copied this from somewhere else, so we should fix that too!)

this should just be `backendconverter`

Reply:
yeah that's in kube-controllers/pkg/converter/pod_converter.go where I got it from.